scripts/prepublish.js

```diff
@@ -9,11 +9,15 @@ shell.echo(chalk.bold(`${packageJson.name}@${packageJson.version}`));
 shell.echo(chalk.gray('\n=> Clean dist.'));
 shell.rm('-rf', 'dist');
 
+const pwd = shell.pwd();
+shell.echo(`pwd = ${pwd}`);
+shell.cd(path.join(__dirname, '..'));
```

Comment:
Babel WAS run inside each folder, using the babel binary from root, but presets from package folders.

This changes so the directory where babel is run from IS root, so presets are taken from there. This will allow us to clean up the babel presets dependencies in packages a bit.